subsys/dfu/dfu_target/src/dfu_target_mcuboot.c

```diff
@@ -36,10 +36,11 @@ LOG_MODULE_REGISTER(dfu_target_mcuboot, CONFIG_DFU_TARGET_LOG_LEVEL);
 static uint8_t *stream_buf;
 static size_t stream_buf_len;
 
-int dfu_ctx_mcuboot_set_b1_file(const char *file, bool s0_active,
-				const char **update)
+int dfu_ctx_mcuboot_set_b1_file(char * const file, bool s0_active,
+				const char **selected_path)
 {
-	if (file == NULL || update == NULL) {
+	LOG_DBG("CURRENTLY ON EXPERIMENTAL");
```

Comment:
Perhaps this was a test message?

Reply:
Shoot! I thought I removed that! Good catch.